File: library/src/main/java/com/alamkanak/weekview/EmptyViewClickListener.java
package com.alamkanak.weekview;

import android.support.annotation.NonNull;

import java.util.Calendar;

public interface EmptyViewClickListener {

    /**
     * Triggered when the users clicks on a empty space of the calendar.
     *
     * @param time: {@link Calendar} object set with the date and time of the clicked position on the view.
     */
    void onEmptyViewClicked(@NonNull Calendar time);

}

File: library/src/main/java/com/alamkanak/weekview/EmptyViewClickListener.kt
package com.alamkanak.weekview

import java.util.*

@FunctionalInterface
interface EmptyViewClickListener {

    /**
     * Called when an empty area of [WeekView] is clicked.
     *
     * @param time A [Calendar] with the date and time of the clicked position
     */
    fun onEmptyViewClicked(time: Calendar)

}
